app/models/user.rb

```diff
@@ -0,0 +1,6 @@
+class User < ApplicationRecord
+	has_and_belongs_to_many :books
```

Comment:
@isabloCh you don't have `book` model/table. You should remove this line for now